client/Cargo.toml

```diff
@@ -17,7 +17,7 @@ is-it-maintained-open-issues = { repository = "https://github.com/hyperledger/ir
 maintenance = { status = "actively-developed" }
 
 [dependencies]
-iroha_config = { version = "=2.0.0-pre-rc.11", path = "../config" }
+iroha_config = { version = "=2.0.0-pre-rc.11", path = "../config", features = [] }
```

Comment:
is the same as `deafult-features == false`?

Reply:
Not exactly. But it doesn't matter for other reasons

https://doc.rust-lang.org/cargo/reference/features.html#feature-unification

Reply:
Yea I forgot to revert this one.